packages/suite/src/views/wallet/send/Outputs/Address.tsx

```diff
@@ -185,6 +190,14 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
                         }),
                     text: translationString('TR_CONVERT_TO_LOWERCASE'),
                 };
+            case 'contract':
+                return {
+                    onClick: () => {
+                        contractAddressWarningDismissed = true;
+                        trigger(inputName);
```

Comment:
what does this do?

Reply:
trigger starts re-validation. same as setValue, but without changing the value. 

https://react-hook-form.com/docs/useform/trigger